src/components/newsFeed/news.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styles from './news.css';
+
+class News extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      readMore: false,
+    };
+  }
+  // eslint-disable-next-line
+  diffHours(dt2, dt1) {
+    let diff = (dt2.getTime() - dt1.getTime()) / 1000;
+    diff /= (60 * 60);
+    return Math.abs(Math.round(diff));
+  }
+
+  render() {
+    const {
+      source, content, timestamp, url, t,
+    } = this.props;
+
+    const hours = this.diffHours(new Date(), new Date(timestamp));
```

Comment:
I would either move to utils function or just use a 1 liner:
https://stackoverflow.com/questions/19225414/how-to-get-the-hours-difference-between-two-date-objects#19225540